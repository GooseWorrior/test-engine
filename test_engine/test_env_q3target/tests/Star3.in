(ab)*

ab
abab
ac
abac
q
(ab)**

ab
abab
ac
abac
q
(ab)*c
c
abc
ababc
abacc
acabc
ac
q
a(bc)*
a
abc
abcbc
ababc
abcba
aba
q
(ab)*(cd)*

ab
cd
abcd
ababcd
abcdcd
ababcdcd
dcd
aba
e
q
((ab)*c*)*

abc
abcabc
ab
abab
cc
cabc
ccab
a
q
