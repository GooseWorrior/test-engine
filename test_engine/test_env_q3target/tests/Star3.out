Star (Concatenation (a,b))
true
true
true
false
false
Star (Star (Concatenation (a,b)))
true
true
true
false
false
Concatenation (Star (Concatenation (a,b)),c)
true
true
true
false
false
false
Concatenation (a,Star (Concatenation (b,c)))
true
true
true
false
false
false
Concatenation (Star (Concatenation (a,b)),Star (Concatenation (c,d)))
true
true
true
true
true
true
true
false
false
false
Star (Concatenation (Star (Concatenation (a,b)),Star (c)))
true
true
true
true
true
true
true
true
false
